run_devstack_integration_tests.sh

```diff
@@ -1,24 +1,26 @@
 #!/bin/bash
 set -e
 
-source /edx/app/edxapp/venvs/edxapp/bin/activate
+source /openedx/venv/bin/activate
+ls
+pwd
 
-cd /edx/app/edxapp/edx-platform
 mkdir -p reports
 
 pip install -r ./requirements/edx/testing.txt
 pip install -r ./requirements/edx/paver.txt
 
+# Installing edx-platform
+pip install -e .
+
 mkdir -p test_root  # for edx
 paver update_assets lms --settings=test_static_optimized
 
 cp test_root/staticfiles/lms/webpack-stats.json test_root/staticfiles/webpack-stats.json
-
 cd /open-edx-plugins
 
-# Installing dev dependencies
-pip install poetry
-poetry install --no-interaction --only dev
```

Comment:
Removed this code of installing dev dependencies as this resulted in a conflict with tutor edx-platform dependencies.